tests/functional/spec/list_collector.spec.js

```diff
@@ -8,6 +8,7 @@ const ListCollectorEditPage = require("../generated_pages/list_collector/list-co
 const ListCollectorRemovePage = require("../generated_pages/list_collector/list-collector-remove.page.js");
 const NextInterstitialPage = require("../generated_pages/list_collector/next-interstitial.page.js");
 const SummaryPage = require("../generated_pages/list_collector/section-summary.page.js");
+const SectionPage = require("../generated_pages/list_collector/section-summary.page");
```

Comment:
This looks odd.

Reply:
We should probably remove the changes to the schema and functional tests now that the schema doesn't need to be modified in this pr.